build_projects/update-dependencies/Program.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.DotNet.VersionTools;
 using Microsoft.DotNet.VersionTools.Automation;
 using Microsoft.DotNet.VersionTools.Dependencies;
+using Microsoft.DotNet.VersionTools.Dependencies.BuildOutput;
```

Comment:
Is this using statement necessary? what uses it?

Reply:
BuildDependencyInfo is defined in this namespace. I had to add it to make the app compile again. Along with a couple other changes.